src/index.js

```diff
@@ -6,14 +6,15 @@ import momentLocalizer from './localizers/moment';
 import globalizeLocalizer from './localizers/globalize';
 import viewLabel from './utils/viewLabel';
 import move from './utils/move';
-import { views } from './utils/constants';
+import { views, navigate } from './utils/constants';
 
 Object.assign(Calendar, {
   setLocalizer,
   globalizeLocalizer,
   momentLocalizer,
   label: viewLabel,
   views,
+  navigate,
```

Comment:
small nit, I'd capitalize this to `Navigate` to be a bit clearer it's a namespace...i should have done that with `views` as well